AndroidSDKCore/src/main/java/com/leanplum/Leanplum.java

```diff
@@ -725,9 +731,10 @@ private static void handleStartResponse(final JSONObject response) {
         // Load the variables that were stored on the device from the last session.
         VarCache.loadDiffs();
 
-        triggerStartResponse(false);
       } catch (Throwable t) {
         Util.handleException(t);
+      } finally {
+        triggerStartResponse(success);
```

Comment:
are we triggering a start response or start request? Maybe its an opportunity to improve our naming?

Reply:
We have the following methods in Leanplum.java:

addStartResponseHandler
removeStartResponseHandler
triggerStartResponse

There is a pattern for the names and If I change one method (triggerStartResponse) then I should change all of them and brake the SDK backward compatibility because the add and remove methods are public. I don't think we should do that at the moment.